src/python/PyWeek2/matching_brackets.py

```diff
@@ -0,0 +1,29 @@
+def group_check(s):
+    brackets = ['(', ')', '[', ']', '{', '}']
+    pairs = ['()', '[]', '{}']
+    list1 = []
+    j = 0
+    for i in range(len(s)):
+        if s[i] in brackets:
+            list1.append(s[i])
+    string_brackets = ''.join(list1)
+    length_string_brackets = len(string_brackets)
+    while length_string_brackets > 0 and ('{}' in string_brackets or '[]' in string_brackets or '()' in string_brackets):
+        current_pair = pairs[j]
+        if current_pair in string_brackets and j < 3:
+            string_brackets = string_brackets.replace(current_pair,'')
+            print string_brackets
+            j = 0
+        else:
+            j += 1
+
+    if len(string_brackets) > 0:
```

Comment:
Because you are using an "if" statement and returning with bools you can simply return with booleans.
You can write something like this: `return len(string_brackets) < 0`

If the expression is true, then it will return with true, else with false.